tripal_chado/src/Plugin/Field/FieldType/ChadoIntegerTypeDefault.php

```diff
@@ -45,6 +62,18 @@ public static function defaultStorageSettings() {
     return $settings;
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public static function generateSampleValue(FieldDefinitionInterface $field_definition) {
+    $value = [];
+
+    $value['record_id'] = 0;
+    $value['value'] = mt_rand(0, 500);
```

Comment:
```suggestion
    $value['value'] = mt_rand(-500, 500);
```
Would it be more robust to include negative values?